src/main/java/frc/robot/commands/ArmCommand.java

```diff
@@ -0,0 +1,144 @@
+    // Use addRequirements() here to declare subsystem dependencies.
+    addRequirements(ArmSubsystem.getInstance());
+  }
+
+
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    if (level=="1") {
+      z = 4;//constants --> first and second z positions - depends on the height of the node we are going for
+      nodeX = 19.382;//the x positiion of the node we are going for
+    } 
+    else if(level=="2") {
+      z = 24;
+      nodeX = 37.132;
+    }
+    else if (level=="3") {
+      z = 35;
+      nodeX = 54.132;
+    }
+    else if (level=="default") {
+      defaultAngles = new double[]{defaultAngle1, defaultAngle2};
+      z = 0;
+      nodeX = 0;
+    }
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+
+    double x = ArmSubsystem.getInstance().setx(0,nodeX);
+    double[] angles = ArmSubsystem.getInstance().calculateAngle(Config.Arm.L1, Config.Arm.L2, x, z);
+    double angle1 = angles[0];
+    double angle2 = angles[1];
+
+    targetAngularDistanceBottomArm = ArmSubsystem.getInstance().getAngularDistance(angle1, Config.Arm.NEO_GEAR_RATIO);
+    targetAngularDistanceTopArm = ArmSubsystem.getInstance().getAngularDistance(angle2, Config.Arm.NEO_GEAR_RATIO);
+    
+    if (level == "3") {
+      // ArmSubsystem.getInstance().setJoint1(angle1); - only used when two motors are involved (testing only)
+      ArmSubsystem.getInstance().setJoint2(angle2); // if level 3 the angle should be 139 degrees, if level 2 the angle should be 74 degrees
+    }
+    else if (level == "2") {
+      ArmSubsystem.getInstance().setJoint1(angle2); // if level 3 the angle should be 59 degrees, if level 2 angle should be 94 degrees
+      // ArmSubsystem.getInstance().setJoint2(angle2); - only used when two motors are involved (testing only)
+    }
+    else if (level == "1") {
+      ArmSubsystem.getInstance().setJoint1(angle2);
+    }
+    else if (level == "default") {
+      ArmSubsystem.getInstance().setDefault(defaultAngles);
+    }
+    // if (ArmSubsystem.getInstance().m_absoluteBottomArmEncoder.getPosition() < targetAngularDistanceBottomArm) {
+    //   ArmSubsystem.getInstance().setJoint1(targetAngularDistanceBottomArm);
+    // }
+    // else {
+    //   ArmSubsystem.getInstance().m_bottomArm.set(0);
+    // }
+
+    // if (ArmSubsystem.getInstance().m_absoluteTopArmEncoder.getPosition() < targetAngularDistanceTopArm) {
+    //   ArmSubsystem.getInstance().setJoint2(targetAngularDistanceTopArm);
+    // }
+    // else {
+    //   ArmSubsystem.getInstance().m_topArm.set(0);
+    // }
+
+    System.out.println(Arrays.toString(angles));
+    System.out.println(targetAngularDistanceBottomArm);
+    System.out.println(ArmSubsystem.getInstance().m_topArm.getEncoder().getPosition());
+    System.out.println(targetAngularDistanceTopArm);
+    System.out.println(ArmSubsystem.getInstance().m_bottomArm.getEncoder().getPosition());
```

Comment:
I'm also not a big fan of printlns in commits, so same question as above: are they needed for the functionality, or can they be taken out?